liblangutil/SemVerHandler.cpp

```diff
@@ -324,3 +332,11 @@ void SemVerMatchExpressionParser::nextToken()
 	++m_pos;
 	m_posInside = 0;
 }
+
+bool SemVerMatchExpressionParser::containPrefixingToken() const
+{
+	if (std::find_if(m_tokens.begin(), m_tokens.end(), TokenTraits::isPragmaOp) != m_tokens.end())
```

Comment:
```suggestion
	if (find_if(m_tokens.begin(), m_tokens.end(), TokenTraits::isPragmaOp) != m_tokens.end())
```
We already have `using namespace std`, so this is redundant.
Also, this method could just return the result of the comparison of the `if` block directly. Maybe, we could not have it at all and just use the comparison directly.